Add Pull Request object to DORA Metrics data collected (#36398)
DataDog/documentation: content/en/dora_metrics/_index.md

### Create custom dashboards

> Build custom dashboards on top of DORA Metrics to analyze your delivery workflow end-to-end, from commits through pull requests to production deployments. For example, compare code review performance across teams to identify which teams are blocked by slow approvals and prioritize where to invest in workflow improvements.

Reviewer: Simplify language for clearer readability:
```suggestion
Build custom dashboards using DORA Metrics to analyze your delivery workflow end-to-end, from commits and pull requests to production deployments. For example, compare code review performance across teams to identify which teams are blocked by slow approvals and prioritize where to invest in workflow improvements.
```
